api_tests/scenario-ee492ced05.yaml

```diff
@@ -0,0 +1,53 @@
+# ********RoostGPT********
+
+# Test generated by RoostGPT for test TDD-based-test using AI Type Azure Open AI and AI Model roost-gpt4-32k
+# 
+# {
+#   background: null,
+#   rule: null,
+#   scenario: 'Update the card balance once payment is collected\n' +
+#     '  Given the system has started collecting payment and the payment is successful\n' +
+#     '  When the client sends a POST request to "/api/creditCard/payment" with the payment status\n' +
+#     '  Then the response status code should be 200\n' +
+#     '  And the card balance should be updated',
+#   title: 'Update the card balance once payment is collected'
+# }
+# 
+
+# ********RoostGPT********
+config:
+  target: "{{ $processEnvironment.API_HOST }}"
+  plugins:
+    expect: {}
+  phases:
+    - duration: 60
+      arrivalRate: 20
+  payload:
+    path: "scenario-ee492ced05.csv"
+    fields:
+      - paymentAmount
+      - creditCardNumber
+    skipHeader: true
+  environments:
+    staging:
+      headers:
+        Authorization: "{{ $processEnvironment.AUTH_TOKEN }}"
+
+scenarios:
+  - name: Update the card balance once payment is collected
+    flow:
+      - log: "System starts collecting payment and the payment is successful"
+      # POST payment details
+      - post:
+          url: "/creditCard/payment"
+          headers:
+            Content-Type: "application/json"
+            Authorization: "{{ $processEnvironment.AUTH_TOKEN }}"
+          json:
+            paymentAmount: "{{ paymentAmount }}"
+            creditCardNumber: "{{ creditCardNumber }}"
+          expect:
+            - statusCode: 200
+            - hasProperty: "balance"
+        think: 1
+      - log: "Card balance should be updated after payment is collected"
```

Comment:
Here output was cut in between. It should do get call as well to fetch the balance